'RuntimeException' @ [21:68] ==> public final fun <init>(p0: (String..String?), p1: (Throwable..Throwable?)): RuntimeException /* = RuntimeException */ defined in kotlin.RuntimeException[TypeAliasConstructorDescriptorImpl]

'message' @ [21:85] ==> value-parameter message: String defined in org.jetbrains.kotlin.util.KotlinFrontEndException.<init>[ValueParameterDescriptorImpl]

'cause' @ [21:94] ==> value-parameter cause: Throwable defined in org.jetbrains.kotlin.util.KotlinFrontEndException.<init>[ValueParameterDescriptorImpl]

'this' @ [26:9] ==> public constructor KotlinFrontEndException(message: String, cause: Throwable) defined in org.jetbrains.kotlin.util.KotlinFrontEndException[ClassConstructorDescriptorImpl]

'getExceptionMessage' @ [26:14] ==> public fun getExceptionMessage(subsystemName: String, message: String, cause: Throwable?, element: PsiElement?): String defined in org.jetbrains.kotlin.util in file exceptionUtil.kt[SimpleFunctionDescriptorImpl]

'message' @ [26:47] ==> value-parameter message: String defined in org.jetbrains.kotlin.util.KotlinFrontEndException.<init>[ValueParameterDescriptorImpl]

'cause' @ [26:56] ==> value-parameter cause: Throwable defined in org.jetbrains.kotlin.util.KotlinFrontEndException.<init>[ValueParameterDescriptorImpl]

'element' @ [26:63] ==> value-parameter element: PsiElement defined in org.jetbrains.kotlin.util.KotlinFrontEndException.<init>[ValueParameterDescriptorImpl]

'cause' @ [26:73] ==> value-parameter cause: Throwable defined in org.jetbrains.kotlin.util.KotlinFrontEndException.<init>[ValueParameterDescriptorImpl]

